app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -0,0 +1,54 @@
+- validate_url ||= 'log_depot_edit'
+
+%div{"ng-show" => "#{ng_show}"}
```

Comment:
I'm kinda confused about this (and the other ruby locals like this). Is it just something to do with the angular-bootstrap? Cause I would much rather see ng-show referencing a scope variable so that when it changes in an angular controller the UI updates.